CodeReview/src/CodeReviewManager.cpp

```diff
@@ -81,7 +85,8 @@ void CodeReviewManager::saveReview(QString newVersion)
 
 }
 
-NodeReviewsList* CodeReviewManager::loadReview(QString newVersion)
+NodeReviewsList* CodeReviewManager::loadReview(QString newVersion, VersionControlUI::DiffSetup& diffSetup,
+																 Visualization::ViewItem* viewItem)
```

Comment:
The names are messed up, I need to have a look at that section again.
Sorry about that.